### ISSUE TITLE
TextSource key

### ISSUE THREAD

**Issue by DavidHaslam** [krisek/HunKar#9]
**hunkar.conf** currently includes:
```
TextSource=http://szentiras.hu/KG
```
and
```
History_2.0=Module rebuilt based on new source with cross-references and section titles (from http://abibliamindenkie.hu/karoli). Description corrected.
```

The **TextSource** key requires updating!

**Comment by krisek** [comment]
Is it possible to define more than one TextSource? Finally Song Of Solomon was included from http://abibliamindenkie.hu/karoli too.

**Comment by DavidHaslam** [comment]
AFAICT, the **TextSource** key may contain more than one URL (separated by a space) as well as any relevant plain text.

**Comment by krisek** [comment]
fixed in 8471ba0a

**Comment by DavidHaslam** [comment]
Rather than having the following:
```
TextSource=http://szentiras.hu/KG, http://abibliamindenkie.hu/karoli
```
for the SWORD module, it would make better sense to have
```
https://github.com/krisek/HunKar/
```

As long as this repo always points to the upstream sources, this would make it easier for any user to trace the source from which the module was actually built.

@krisek 

Please remember to update this key whenever the .conf file is edited.
```
SwordVersionDate=2020-08-21
```
